packages/features/eventtypes/components/AddMembersWithSwitch.tsx

```diff
@@ -267,6 +270,15 @@ function AddMembersWithSwitch({
             }}
             onInactive={onAssignAllTeamMembersInactive}
           />
+          {!assignRRMembersUsingSegment ? (
```

Comment:
Currently only render when were not using RR member assigment for attributes as the teamMember logic comes from internal and gets pretty complex. Will do this in a follow up to support weights on attribute assignees too